scripts/depcheck/missing.ts

```diff
@@ -126,6 +126,13 @@ const IGNORE_MATCHES = {
   ],
   nest: ['semver'],
   'make-angular-cli-faster': ['@angular/core'],
+  'react-native': [
+    '@babel/core',
```

Comment:
I think we should actually have these as dependencies?

Reply:
removed, no longer need them.